lib/supa/commands/attribute.rb

```diff
@@ -2,8 +2,20 @@ module Supa
   module Commands
     class Attribute < Supa::Command
       def represent
+        return if hide?
+
         @tree[@name] = value
       end
+
+      private
+
+      def hide?
+        value.is_a?(NilClass) && hide_when_nil?
```

Comment:
I did it like that initially by it violates [reek's check ](https://github.com/troessner/reek/blob/master/docs/Nil-Check.md).